### PR TITLE
feat: add Henez tesnet config

### PR DESCRIPTION
**Pull request by louis-henez**
### Description

Add Henez testnet chain config

### Backward compatibility
Yes

### Testing
Yes

### CODE REVIEW

**Comment by louis-henez** [comment]
There is no second rpc url for the testnet env so that I can't add to the metadata as a fallback

File: chains/heneztestnet/logo.svg [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<stop offset="1" stop-color="white" stop-opacity="0.15"/>
+</linearGradient>
+<clipPath id="clip0_19632_62123">
+<rect width="450" height="450" fill="white"/>
+</clipPath>
+<clipPath id="clip1_19632_62123">
+<rect width="402.142" height="247.806" fill="white" transform="translate(-10.8594 218.873) rotate(-30)"/>
+</clipPath>
+</defs>
+</svg>
```
Comment (paulbalaji) [inline chains/heneztestnet/logo.svg:26]: ran through svg minifier https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="450" height="450" fill="none"><g clip-path="url(#a)"><path fill="#000" d="M0 0h450v450H0z"/><g clip-path="url(#b)"><path fill="#F5F6F8" d="M252.081 268.912c24.427-14.102 32.834-45.27 18.779-69.615-14.055-24.344-45.251-32.646-69.678-18.543-24.427 14.102-32.835 45.27-18.78 69.615 14.056 24.344 45.252 32.646 69.679 18.543Z"/><path fill="url(#c)" d="M340.576 187.508c14.835 57.308-11.895 111.217-56.403 136.978-51.649 29.811-116.685 15.937-151.294-32.265a120.283 120.283 0 0 1-4.728-7.126c3.558-1.933 7.113-3.856 10.646-5.784a104.346 104.346 0 0 0 5.19 7.601c27.296 36.36 74.807 49.846 116.563 35.205a96.982 96.982 0 0 0 15.592-6.961c.377-.218.762-.421 1.146-.643.28-.161.57-.32.85-.481l-.013-.021a92.03 92.03 0 0 0 4.74-2.968c36.556-23.454 54.146-72.002 39.053-112.772 9.092 45.747-12.825 87.718-48.193 108.119l-.008-.014c-41.372 23.886-94.093 13.231-122.627-24.789a94.347 94.347 0 0 1-4.481-6.536c4.191-2.29 8.34-4.555 12.442-6.803 9.198 14.865 23.608 26.6 40.24 32.44 57.919 20.74 117.541-30.536 102.227-90.639 5.099 40.201-21.522 78.493-61.886 85.482-31.013 5.702-63.029-10.188-77.771-37.916l-3.298 1.997c-17.592 10.528-36.368 22.361-53.889 32.996-15.378 9.51-38.64 23.877-53.568 33.554 4.698-2.397 9.358-4.856 14.029-7.283a7170.274 7170.274 0 0 0 55.216-29.568 128.11 128.11 0 0 0 5.318 8.066c17.837 24.847 44.216 42.117 74.285 48.635a124.092 124.092 0 0 0 61.292-2.191 119.466 119.466 0 0 0 24.332-10.03 125.468 125.468 0 0 0 3-1.676l-.012-.021c48.1-28.152 73.946-88.028 51.998-144.606l.012.02Z"/><path fill="url(#d)" d="M385.688 132.181a5858.275 5858.275 0 0 0-52.764 28.171 124.484 124.484 0 0 0-13.691-18.489c-18.708-20.854-44.349-34.847-72.202-39.415-16.207-2.652-32.747-2.011-48.601 1.722a19.1 19.1 0 0 0-.687.165c-.491.116-.979.24-1.473.368a119.568 119.568 0 0 0-28.651 11.196c-.987.532-1.959 1.075-2.929 1.635l.012.021c-48.098 28.141-73.958 88.025-52.003 144.599-14.835-57.307 11.899-111.209 56.414-136.974 47.061-27.162 107.415-17.656 143.52 22.591a115.99 115.99 0 0 1 12.496 16.803c-3.556 1.923-7.107 3.853-10.651 5.778a104.833 104.833 0 0 0-5.19-7.602 101.946 101.946 0 0 0-8.822-10.265c-28.351-28.864-70.506-38.009-107.758-24.92a97.232 97.232 0 0 0-15.427 6.875 4.642 4.642 0 0 1-.304.166c-.308.178-.619.348-.926.526-.294.169-.591.331-.891.505l.012.021a92.729 92.729 0 0 0-4.747 2.972c-36.556 23.454-54.149 71.995-39.057 112.764-9.092-45.746 12.837-87.714 48.204-108.115l.008.014c37.357-21.568 84.456-15.23 114.55 15.412a94.28 94.28 0 0 1 8.07 9.381 93.998 93.998 0 0 1 4.481 6.536 9609.09 9609.09 0 0 0-12.442 6.803c-9.197-14.866-23.608-26.6-40.236-32.434-57.915-20.732-117.536 30.544-102.231 90.633-5.095-40.194 21.537-78.484 61.89-85.476 31.009-5.708 63.029 10.189 77.767 37.91l3.298-1.997c16.851-10.071 35.721-21.95 52.505-32.169 17.194-10.605 35.181-21.788 52.158-32.768-4.578 2.328-9.124 4.711-13.683 7.074l-.019-.017Z"/></g></g><defs><linearGradient id="c" x1="247.78" x2="297.124" y1="336.028" y2="209.415" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"/><stop offset="1" stop-color="#fff" stop-opacity=".15"/></linearGradient><linearGradient id="d" x1="201.367" x2="152.086" y1="112.771" y2="243.324" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"/><stop offset="1" stop-color="#fff" stop-opacity=".15"/></linearGradient><clipPath id="a"><path fill="#fff" d="M0 0h450v450H0z"/></clipPath><clipPath id="b"><path fill="#fff" d="M-10.86 218.873 337.407 17.802l123.903 214.606L113.044 433.48z"/></clipPath></defs></svg>
```

File: chains/heneztestnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://henez-testnet.explorer.caldera.xyz/api/
```
Comment (paulbalaji) [inline chains/heneztestnet/metadata.yaml:3]:
```suggestion
  - apiUrl: https://henez-testnet.explorer.caldera.xyz/api/eth-rpc
```

File: chains/heneztestnet/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    family: blockscout
+    name: Henez Testnet Explorer
+    url: https://explorer.henez.fi/
```
Comment (paulbalaji) [inline chains/heneztestnet/metadata.yaml:6]:
```suggestion
    url: https://explorer.henez.fi
```

File: chains/heneztestnet/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+chainId: 91111
+deployer:
+  name: Henez Foundation
+  url: https://www.henez.fi/
```
Comment (paulbalaji) [inline chains/heneztestnet/metadata.yaml:10]:
```suggestion
  url: https://www.henez.fi
```

**Comment by louis-henez** [comment]
Updated, thank you for your review @paulbalaji

**Pull request merged**